python/pyproject.toml

```
@@ -288,3 +281,47 @@ ignore_missing_imports = true
 
 [tool.coverage.run]
 source = ['pyiceberg/']
+
+[tool.ruff]
+src = ['pyiceberg/','/tests']
```

Are you sure that these paths are relative to the python repository? 
```suggestion
src = ['pyiceberg/', 'tests/']
```